sonic-xcvrd/xcvrd/xcvrd_utilities/port_mapping.py

```diff
@@ -138,6 +139,8 @@ def handle_port_update_event(sel, asic_context, stop_event, logger, port_change_
                 (key, op, fvp) = port_tbl.pop()
                 if not key:
                     break
+                if key.startswith((backplane_prefix(), inband_prefix(), recirc_prefix())):
```

Comment:
Looks good, suggest to have a function defined eg: validate_port which will have this logic.

Reply:
Ok.  I have created validate_port function to handle it as suggest. Also add the modify the test case to test the change